docs/articles/nunit/writing-tests/attributes.md

```diff
@@ -16,6 +16,7 @@ This table lists all the attributes supported by NUnit.
 | [DefaultFloatingPointTolerance Attribute](attributes/defaultfloatingpointtolerance.md) | Indicates that the test should use the specified tolerance as default for float and double comparisons. |
 | [Description Attribute](attributes/description.md)         | Applies descriptive text to a Test, TestFixture or Assembly. |
 | [Explicit Attribute](attributes/explicit.md)            | Indicates that a test should be skipped unless explicitly run. |
+| [FixtureLifeCycle Attribute](attributes/fixturelifecycle.md)  | Specifies the lifecycle of a Fixture allowing a new instance of a test fixture to be constructed for each test. |
```

Comment:
I would say "each test case" to be absolutely clear. We talk about there being a hierarchical tree of tests and fixtures are only created for the bottom level, right?

Reply:
I would consider not capitalizing the word 'fixture' because it's not being used as a proper noun.

Reply:
Maybe we could also comment that using this attribute is desirable if there is test case parallelism, if there's room to say it.